src/main/kotlin/controllers/worldRenderer/Renderer.kt

```diff
@@ -812,12 +812,13 @@ class Renderer @Inject constructor(
     private fun drawTiles() {
         modelBuffers.clear()
         modelBuffers.targetBufferOffset = 0
-        for (x in 0 until scene.radius * REGION_SIZE) {
-            for (y in 0 until scene.radius * REGION_SIZE) {
-                var tile: SceneTile? = scene.getTile(0, x, y)
-                if (tile != null) {
-                    drawTile(tile)
-                }
+        for (z in 0 until REGION_HEIGHT) {
```

Comment:
Any idea how to fix the height of the second floor walls on region id 14642?

Reply:
Hey! Do you mean these?

![image](https://user-images.githubusercontent.com/58113606/186955253-ae3c9d98-022d-4be8-976d-4abdfacb38b6.png)


No idea actually.

DO you think we could just merge this MR and then resolve this later? At least multiple heights would work.